vlib/datatypes/README.md

```diff
@@ -26,4 +26,5 @@ println(stack)
 - [x] Stack (LIFO)
 - [x] Queue (FIFO)
 - [x] Min heap (priority queue)
+- [ ] Set
```

Comment:
Should be `[x]`, otherwise LGTM

Reply:
it isn't implemented though, I added this as a hopeful/potential datatype to implement

Reply:
Okay got it 👍